### ISSUE TITLE
Models/README.md has broken links

### ISSUE THREAD

**Issue by emackey** [KhronosGroup/glTF-Sample-Assets#36]
https://github.com/KhronosGroup/glTF-Sample-Assets/blob/main/Models/README.md

This file has paths with parent folder references (`../Models.md`) to files that are in the same folder as itself.

**Comment by cx20** [comment]
I think it would be best to modify it as follows to make it consistent.

As Is:
https://github.com/KhronosGroup/glTF-Sample-Assets/blob/43380bb399f5c86acb541ee75af4b5905b6926c5/Models/README.md#L5-L11

To Be:
```
* [Showcase](./Models-showcase.md) (#showcase) for models that are showcased in Khronos publicity. These are usually complex models with high visual quality.
* [Complete](./Models.md) for a complete list of all models.
* [Testing](./Models-testing.md) (#testing) for models intended to be used for testing of viewers, converts, and other software systems.
* [Core Only](./Models-core.md) (#core) for models that only use glTF Core V2.0 features and capabilities (no extensions).
* [Video Tutorials](./Models-video.md) (#video) for models used in any glTF Tutorial video.
* [Written Tutorials](./Models-written.md) (#written) for models used in a written glTF Tutorial.
* [Issues](./Models-issues.md) (#issues) for models with one or more issues that need to be resolved.
```

**Comment by cx20** [comment]
https://github.com/KhronosGroup/glTF-Sample-Assets/blob/43380bb399f5c86acb541ee75af4b5905b6926c5/README.md?plain=1#L65

As Is : `glTF-Sample-Models`
To Be : `glTF-Sample-Assets`

https://github.com/KhronosGroup/glTF-Sample-Assets/blob/43380bb399f5c86acb541ee75af4b5905b6926c5/Models/model-index.json#L76

As Is : `screenshot.png`
To Be : `screenshot.jpg`

https://github.com/KhronosGroup/glTF-Sample-Assets/blob/43380bb399f5c86acb541ee75af4b5905b6926c5/Models/Models.md#L27

As Is : `screenshot.png`
To Be : `screenshot.jpg`

**Comment by DrX3D** [comment]
@cx20: Your above comments are appreciated and have been fixed in PR #87.
@emackey: Your note on the Models/README links has also been fixed in PR #87.

That PR is schedule for installation on 4 Jan (sooner if there are sufficient reviews of the new models).

**Comment by emackey** [comment]
This was fixed a while ago.
